packetbeat/protos/dns/dns.go

```diff
@@ -292,14 +291,14 @@ func (dns *dnsPlugin) ConnectionTimeout() time.Duration {
 }
 
 func (dns *dnsPlugin) receivedDNSRequest(tuple *dnsTuple, msg *dnsMessage) {
-	debugf("Processing query. %s", tuple.String())
+	logp.Debug("dns", "Processing query. %s", tuple)
```

Comment:
Could we store a `*logp.Logger` into the dnsPlugin? IIRC logp.Debug is deprecated in favor of Logger instance methods.

Reply:
How does that interact with `logp.DevelopmentSetup`? I don't see a way to obtain the same behaviour with a `logp.Logger` that is requested in dns_test.go's newDNS.